src/components/inputs/filled-icon-input/index.jsx

```diff
@@ -0,0 +1,33 @@
+import { TextField, InputAdornment } from "@mui/material";
+
+const FilledIconInput = ({
+  icon = null,
+  placeHolder = "place holder",
+  isIconStart = true,
+  text = null,
```

Comment:
For text prop, you are defaulting it to null. This is okay but you might want to consider setting it to an empty string '' to avoid any potential null reference errors

Reply:
Yes, I will change it, thanks.